docs/portal_config.md

```diff
@@ -182,6 +182,7 @@ Below is an example, with inline comments describing what each JSON block config
     "explorerFilterValuesToHide": ["array of strings"], // optional, Values set in array will be hidden in guppy filters. Intended use is to hide missing data category from filters, for this it should be set to the same as `missing_data_alias` in Guppy server config
     "studyRegistration": true, // optional, whether to enable the study registration feature
     "workspaceRegistration": true, // optional, whether to enable the workspace registration feature
+    "workspaceTokenServiceRefreshTokenAtLogin": true, // optional, whether to refresh the WTS token directly at portal login (recommended mode). If not set, this refresh happens only when the user enters the workspace section of the portal (default/old/previous mode).
```

Comment:
I feel like the default here should be `false` if we're trying to be backwards compatible w/o needing to update config. That way, people that want to take advantage of this can turn it on in there config but there's no change to those who want the old previous mode

Reply:
Yes, the default is "not set", so `isEnabled('workspaceTokenServiceRefreshTokenAtLogin')` should return `false`.